### PR TITLE
Tweak readme (and yes, I meant to open this PR ;)

### PR DESCRIPTION
**Pull request by qwertychouskie**
Simplification, and formatting fixes.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -47,15 +49,11 @@ For technical questions regarding the SDK, please visit the FTC Technology forum
...
-Version 3.2 (built on 17.08.02)
-
-This version of the software introduces the "OnBot Java" Development Tool.  Similar to the FTC Blocks Development Tool, the FTC OnBot Java Development Tool allows a user to create, edit and build op modes dynamically using only a Javascript-enabled web browser.
-
-The OnBot Java Development Tool is an integrated development environment (IDE) that is served up by the Robot Controller.  Op modes are created and edited using a Javascript-enabled browser (Google Chromse is recommended).  Op modes are saved on the Robot Controller Android device directly.  
-
-The OnBot Java Development Tool provides a Java programming environment that does NOT need Android Studio.
+### Version 3.2 (built on 17.08.02)
 
+This version of the software introduces the "OnBot Java" Development Tool.  Similar to the FTC Blocks Development Tool, the FTC OnBot Java Development Tool allows a user to create, edit and build op modes dynamically using only a web browser, and does NOT need Android Studio.
```
Comment (magneticflux-) [inline README.md]: You might want to keep the information about Javascript being required there, on the off chance someone tries to use it using something ridiculous (or they just have Javascript disabled with a whitelist).

File: README.md [hunk 2]
```diff
@@ -12,7 +12,9 @@ It is important to note that this repository is large and can take a long time a
 
 * If you are a git user, *FIRST* recommends that you use the --depth command line argument to only clone the most current version of the repository:
 
-<p>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;git clone --depth=1 https://github.com/ftctechnh/ftc_app.git</p>
+```bash
+git clone --depth=1 https://github.com/ftctechnh/ftc_app.git</p>
```
Comment (MidLifeXis) [inline README.md]: Should this line include the ```</p>```?

Comment (qwertychouskie) [reply README.md]: Fixed, thanks.

File: README.md (hunk 1 above)
Comment (qwertychouskie) [reply README.md]: OK, fixed.

**Comment by WardBenjamin** [comment]
I still think that the Google Chrome recommendation should be left in, since it's by far the most common browser and easiest to support from FIRST's end (especially in JS/CSS features).

**Comment by cmacfarl** [comment]
The release process, as a general rule, does not allow for direct commits to ftc_app. If there are grammatical and/or changes you'd like to see you may open an issue describing them and the request will be evaluated. Thank you.

**Pull request closed without merging**